python/integration_test.py

```diff
@@ -60,8 +62,12 @@ def test_translator_and_machine(golden, caplog):
 
     # Создаём временную папку для тестирования приложения.
     with tempfile.TemporaryDirectory() as tmpdirname:
+        # Определяем язык golden теста. По умолчанию предполагается язык Brainfuck
+        lang = golden.get("lang", "bf")
+        assert lang == "bf" or lang == "asm"
+
         # Готовим имена файлов для входных и выходных данных.
-        source = os.path.join(tmpdirname, "source.bf")
+        source = os.path.join(tmpdirname, "source." + lang)
```

Comment:
```suggestion
        source = os.path.join(tmpdirname, "source")
```

No body see this files...